packages/schema/src/Schema.ts

```diff
@@ -1933,7 +1971,7 @@ export const optional: {
       | (Types.Has<Options, "as"> extends true ? option_.Option<A> : A)
       | (Types.Has<Options, "as" | "default" | "exact"> extends true ? never : undefined),
       never,
-      "?:",
+      Types.Has<Options, "onNoneEncoding"> extends true ? ":" : "?:",
```

Comment:
the same applies here, it should be `"?:"` as before

Reply:
My bad, sorry! I'm fixing that quickly
Why not having define types that are re-used in both overloads? It feels a bit painful to have to update both every time but there might be some reasons I don't know of.

Reply:
yeah, now that the type is quite big, it makes even more sense to define the type only once